Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -148,7 +148,8 @@ class AppSettingsTableViewController: SettingsTableViewController, FeatureFlagsP
 
         if #available(iOS 14.0, *) {
             settings += [
-                SettingSection(footerTitle: NSAttributedString(string: String.DefaultBrowserCardDescription), children: [DefaultBrowserSetting()])
+                SettingSection(footerTitle: NSAttributedString(string: String.FirefoxHomepage.HomeTabBanner.EvergreenMessage.DefaultBrowserDescription),
```

Comment:
I forgot to mention this, but, technically, if you're passing in a thing that's already expecting a `String` type, you don't have to do `String.a.bunch.of.stuff`. You can omit the `String.` part, which is pretty great.

Total nit for now though and there's no real need to make updates.